spec/models/watcher_spec.rb

```diff
@@ -25,10 +27,4 @@
   %w[5w 30d 800h].each do |value|
     it { is_expected.not_to allow_value(value).for(:time_window) }
   end
-
-  describe '.conditions' do
-    subject(:conditions) { watcher.conditions }
-
-    it { is_expected.to be_a Hash }
-  end
```

Comment:
I've moved this with some explanation to the shared specs in [spec/support/watcher_behavior.rb](https://github.com/GSA/search-gov/pull/1088/files#diff-bd7b50568935b1eeba50c0c68135b23e8c4492a960d59da978bb78a8a309899d)